docs/autogen.py

```diff
@@ -359,14 +366,17 @@ def copy_examples():
 
 
 def generate():
+    sources_dir = keras_dir / 'docs/sources'
```

Comment:
Prefer `os.path.join` instead of `/`, everywhere (will work on Windows, and generally is cleaner)